processmanager/helpers.go

```diff
@@ -79,3 +89,32 @@ type FileIDMapper interface {
 	// Set adds a mapping from the 64-bit file ID to the 128-bit file ID.
 	Set(pre host.FileID, post libpf.FileID)
 }
+
+// extractContainerID returns the containerID for pid if cgroup v2 is used.
+func extractContainerID(pid libpf.PID) (string, error) {
```

Comment:
It would be great to have tests like these:
https://github.com/open-telemetry/opentelemetry-ebpf-profiler/blob/7d42302de4798b9ee6b8d704b509c014aedf7000/reporter/util_test.go#L14-L56